pkgs/by-name/se/serd/package.nix

```diff
@@ -15,7 +15,7 @@
 
 stdenv.mkDerivation rec {
```

Comment:
change to finalAttrs pattern while we're at it?

Reply:
Changed as [serd: access to version via finalAttrs](https://github.com/NixOS/nixpkgs/pull/375252/commits/95f46cf70bdc142b0b7eadfe91b2e8bf8c389aff)